lib/codit/outbreak.py

```diff
@@ -57,6 +76,8 @@ def simulate(self):
             self.society.manage_outbreak(self.pop)
             self.pop.attack_in_groupings(self.group_size)
             self.record_state()
+        # close plt after cameras have been saved to avoid showing single heatmap
```

Comment:
Could we encapsulate as much as possible of the animation as part of a separate Object which the OutbreakRecorder optionally calls? I would not want this to be done every time as it demands memory and cpu time.

Reply:
Yes, good idea. I will encapsulate it.